plugins/Live/templates/getLastVisitsStart.twig

```diff
@@ -85,7 +85,7 @@
 {{ action.serverTimePretty }}
 {% if action.timeSpentPretty is defined %}{{ 'General_TimeOnPage'|translate }}: {{ action.timeSpentPretty|raw }}{% endif %}
 {%- endset %}
-                                    <img src="plugins/Live/images/file{{ col }}.png" title="{{ action.url|e('html_attr') }}&mdash;{{- title -}}"/>
+                                    <img src="plugins/Live/images/file{{ col }}.png" title="{{ action.url|e('html_attr') }}<br/>{{- title -}}"/>
```

Comment:
Works but when I tested it I didn't have a URL set and then an empty line appears. that should be fine though I suppose (but maybe easy to fix?). 
![image](https://user-images.githubusercontent.com/273120/51808558-5b461a00-22fa-11e9-869f-c2e084c3e32a.png)